src/index.rs

```diff
@@ -16,7 +16,7 @@ use crate::{
 use tantivy as tv;
 use tantivy::{
     directory::MmapDirectory,
-    schema::{NamedFieldDocument, Term, Value},
+    schema::{NamedFieldDocument, Term, OwnedValue as Value},
```

Comment:
Reminder: remove alias

Reply:
I'm not going to change these alias in this PR. If we want to change it, it can come in a future PR.